robonomics/io/src/source/virt.rs

```diff
@@ -95,8 +98,8 @@ pub fn launch(remote: String) -> impl Stream<Item = (String, String, bool)> {
             if let Ok(event) = NewLaunchEvent::<Robonomics>::decode(&mut &raw.data[..]) {
                 let _ = sender
                     .send((
-                        event.sender.to_string(),
-                        event.robot.to_string(),
+                        event.sender.to_ss58check_with_version(Ss58AddressFormat::RobonomicsAccount),
```

Comment:
Could you move it to an argument? It's better to be able to change the address format from CLI or source code.